deepchecks/vision/checks/distribution/image_dataset_drift.py

```diff
@@ -22,6 +22,8 @@
 __all__ = ['ImageDatasetDrift']
 
 DEFAULT_IMAGE_PROPERTIES = ['aspect_ratio',
+                            'blur',
```

Comment:
Is there DEFAULT_IMAGE_PROPERTIES  for property drift as well? Maybe make this defaults list a common thing rather than duplicated X 3